tests/unit/nupic/research/spatial_pooler_unit_test.py

```diff
@@ -1191,11 +1189,6 @@ def testInitPermanence1(self):
     self.assertGreater(numcon, 0)
     self.assertLess(numcon, sp._numInputs)
 
-    minThresh = sp._synPermActiveInc / 2.0
-    connThresh = sp._synPermConnected
-    self.assertEqual(numpy.logical_and((perm >= minThresh),
-                                       (perm < connThresh)).any(), True)
```

Comment:
i think both should no longer be true, since they were testing whether the permanences are initialized according to the old rule that involved one fourth of synPermActiveInc.